docs/README.md

```diff
@@ -27,10 +34,14 @@ export PATH=$PATH:/Library/TeX/Distributions/.DefaultTeX/Contents/Programs/texbi
 
 ### Building Docs:
 ```bash
-python setup.py build_ext --inplace
```

Comment:
did this not work for you? I think this worked for me last time.

Reply:
It didn't work for me, for some reason I had to go to the root and run ./build.sh, then go back down to the docs folder.

Reply:
Interesting. I agree you can do the source build this way to get a similar result, but the inplace build should do it and worked for me in the past.